packages/charts/examples/components/demo/advanced-usage/legend/legend-metadata/legend-metadata-example.component.ts

```diff
@@ -0,0 +1,156 @@
+import { Component, OnInit } from "@angular/core";
+import {
+    areaGrid,
+    AreaRenderer,
+    Chart,
+    ChartAssist,
+    IAccessors,
+    IAreaAccessors,
+    IChartAssistSeries,
+    IChartSeries,
+    ILineAccessors,
+    IXYScales,
+    LineAccessors,
+    LinearScale,
+    LineRenderer,
+    stackedArea,
+    stackedAreaAccessors,
+    TimeScale, XYAccessors, XYRenderer,
+} from "@nova-ui/charts";
+import moment from "moment/moment";
+
+@Component({
+    selector: "area-chart-stack-example",
+    templateUrl: "./legend-metadata-example.component.html",
+})
+export class LegendMetadataExampleComponent implements OnInit {
+    public chart: Chart;
+    public chartAssist: ChartAssist;
+    public avgSeries: IChartAssistSeries<ILineAccessors>
+
+    public ngOnInit() {
+        // areaGrid returns an XYGrid configured for displaying an area chart's axes and other grid elements.
+        this.chart = new Chart(areaGrid());
+        // ChartAssist will use the preprocessor to stack the series' numeric values on the same progression domain
+        this.chartAssist = new ChartAssist(this.chart, stackedArea);
+
+        // Stacked Area accessors let the renderer know how to access x and y domain data respectively from a chart's input data set(s).
+        const accessors =  stackedAreaAccessors();
+        // 'y1' defines access to the numeric values we want to visualize.
+        // The items in the data array of this example have a property named 'value',so we'll use that.
+        accessors.data.y1 = (d) => d.value;
+
+        // The area renderer will make the chart look like a area chart.
+        const renderer = new AreaRenderer();
+
+        // In case of a area chart, the scale definitions are flexible.
+        // This example demonstrates a scenario with time on the X scale and a numeric value on the Y scale.
+        const scales: IXYScales = {
+            x: new TimeScale(),
+            y: new LinearScale(),
+        };
+
+        const averageData = getAverage(getData());
+        // We are using a different renderer so the metadata does not effect the domain
+        const noopRenderer = new XYRenderer({ignoreForDomainCalculation: true});
+        // Here we create an accessor for our average metadata
+        const avgAccessors = new XYAccessors();
+        // This is so the legend knows the value for the y
+        avgAccessors.data.y = (d) => d.value;
+        // Same as above we are setting the numeric value we want to visualize
+        avgAccessors.data.y1 = avgAccessors.data.y;
+        // Setting the color of the series to transparent so it doesn't render
+        avgAccessors.series.color = () => "var(--nui-color-bg-transparent)";
```

Comment:
Maybe we could change it to this and use it in the template since we no longer draw the series on the chart:

```suggestion
        avgAccessors.series.color = () => "var(--nui-color-bg-mid_dark)";
```

Reply:
Actually, maybe modifying the accessor is unnecessary and the color can just be used directly in the template.